README.md

```diff
@@ -115,4 +121,33 @@ export default {
 };
 ```
 
+### `onBeforeRouteUpdate` and `onBeforeRouteLeave`
+
+Adds a navigation guard that triggers whenever the component for the current location is about to be left. Similar to `beforeRouteUpdate` and `beforeRouteLeave` but can be used in any component. The guard is removed when the component is unmounted.
+
+```js
+import { onBeforeRouteLeave, onBeforeRouteUpdate } from 'vue-router-compositions'
```

Comment:
the naming convention for hooks should be "useX", so please just provide the hooks names using that convention. 🙏

Reply:
Not for hooks... the hooks coming from `vue-router` itself are called `onBeforeRouteLeave` and `onBeforeRouteUpdate`, as can be seen [on their documentation](https://router.vuejs.org/guide/advanced/composition-api.html#navigation-guards).

I named them the same because they are, basically, drop ins for people who cannot migrate to vue 3 + vue-router 4 yet.

Reply:
Got you 👍